### PR TITLE
#5942 feat(refactor): replace rem() functions with CSS variables

### PR DESCRIPTION
**Pull request by chriddell**
Implements https://github.com/wellcometrust/corporate/issues/5942

### CODE REVIEW

File: src/components/PageHeader/_page-header.scss [hunk 1]
```diff
@@ -1,18 +1,10 @@
 .title-banner__container {
-  // padding: rem(var(--text-base-size-px) * 2) 0;
-  padding: rem(32) 0;
+  padding-left: var(--space-xl);
```
Comment (creido-welly) [inline src/components/PageHeader/_page-header.scss]: --space-xl will change size depending on the breakpoint (based on spacings in Wellcome Design System - https://zpl.io/bJWJNgW)

is that what you need?

if these values are static or differ in any way then use multiples of --space-unit

Comment (chriddell) [reply src/components/PageHeader/_page-header.scss]: Yep, --space-xl is intended.

Here's the settings for it:

```css
:root {
  // 32px
  --space-xl: calc(4 * var(--space-unit));

  @include mq(sm) {
    // 48px
    --space-xl: calc(6 * var(--space-unit));
  }

  @include mq(md) {
    // 64px
    --space-xl: calc(8 * var(--space-unit));
  }
}

Comment (creido-welly) [reply src/components/PageHeader/_page-header.scss]: 👍 great, just checking

File: src/components/PageHeader/_page-header.scss [hunk 2]
```diff
@@ -28,34 +20,22 @@
...
+  padding-bottom: var(--space-xl);
   text-align: center;
 
-  @include mq(sm) {
-    padding-bottom: rem(32);
-  }
-
-  @include mq(md) {
-    padding-bottom: rem(64);
+  @include mq($from: sm, $until: md) {
```
Comment (creido-welly) [inline src/components/PageHeader/_page-header.scss]: I'm guessing this is an override for --space-xl. Title banner is one of the more recent components which would almost certainly use the new spacings so I'd remove this and let the variable height work as normal (unless it looks wrong?)

Comment (chriddell) [reply src/components/PageHeader/_page-header.scss]: Sounds good, I'll do a quick visual check to make sure

File: src/components/ResultItem/_result-item.scss [hunk 3]
```diff
@@ -1,13 +1,13 @@
 .result-item {
   border-top: 1px solid var(--colour-concrete-10);
-  padding-bottom: rem(25);
-  padding-top: rem(25);
+  padding-bottom: calc(3.125 * var(--space-unit));
```
Comment (creido-welly) [inline src/components/ResultItem/_result-item.scss]: All good here but might be worth getting Jason's thoughts on the result item spacings. If we can introduce the new spacing standard quickly and easily then it'll bring things into step with his latest work.

Comment (chriddell) [reply src/components/ResultItem/_result-item.scss]: Agreed - will do

**Review by creido-welly — APPROVED** [review]
Code changes approved - refer older spacings of search result list items to Jason for comment

**Pull request merged**